### DISCUSSION TITLE
Google-Analytics

### DISCUSSION THREAD

**Discussion by mawoka-myblock** [appwrite/appwrite#1579] [General]
Why are you using Google-Analytics in the Dashboard, if you want to build an alternative to Google? Isn't [Plausible](https://plausible.io) enough?

**Comment by eldadfux** [comment]
We have actually testing that out right now. You can see we're running both on appwrite.io before we can complete a switch.

**Reply by mawoka-myblock** [reply]
Yes, please do that! I just noticed, your website has even more trackers on it! Hotjar, Facebook and Google (twice)!

**Reply by mawoka-myblock** [reply]
I think it's really unfair to track the user on their own server with no-optout Google-Analytics!

**Reply by eldadfux** [reply]
We actually do provide an opt-out. Use the Do-Not-Track header flag on your browser, and it will be off.

**Reply by mawoka-myblock** [reply]
> We actually do provide an opt-out. Use the Do-Not-Track header flag on your browser, and it will be off.

That's good, but you could write that in the docs somewhere or something else. The easiest and most ethical way: Remove the stupid Google-Analytics!

**Reply by eldadfux** [reply]
Yeh, we'll discuss this during the weekly team meeting and think where we can mention that in the docs more clearly.
We definitely don't intend to be intrusive and we only collect usage stats to make the product better. We DON'T collect any projects or users data. Thanks for the feedback!

**Reply by mawoka-myblock** [reply]
> Yeh, we'll discuss this during the weekly team meeting and think where we can mention that in the docs more clearly.
> We definitely don't intend to be intrusive and we only collect usage stats to make the product better. We DON'T collect any projects or users data. Thanks for the feedback!

I can really understand why you do that and my problem isn't that you want to know errors loading-times etc., BUT my problem is that Google has all the data, too! So I would complain if you'd use something which isn't owned by Big-Tech

**Reply by eldadfux** [reply]
Yeh, we'll examine plausible for the self-hosted dashboard as well, we just need to make sure that by doing that we don't hit their domain limits.

**Reply by mawoka-myblock** [reply]
That's definitely ok! I really hope you won't encounter any issues setting up plausible!

**Reply by vierzwei** [reply]
I installed appwrite and tried it for a project, but we decided against it, due to the unclear concept of privacy / calling home from our server / having google analytics run on the admin dashboard. The appwrite.io privacy policy sounds frightening for use in a DSGVO sensible context. It would definitely help to have some clearer statements & env variables to control the behavior.
